sonar-project.properties

```diff
@@ -0,0 +1,18 @@
+# required metadata
+sonar.projectKey=SFTPAdapter
+sonar.host.url=https://sonarq.systelab.net
+sonar.login=2ff67f60569e363cb08fae89acc70009882b2ca1
+sonar.projectName=SFTPAdapter
+sonar.language=c++
+
+# path to source directories (required)
+sonar.sources=SFTPClientCLI,SftpLibsshAdapter,SFTPAdapterInterface
+# sonar.tests=
+
+# paths to the reports
+sonar.cxx.cppcheck.reportPath=report/SFTPAdapter_cppcheck.xml
+sonar.cxx.xunit.reportPath=report/ResultPackageReport*_Release.xml
+sonar.cxx.coverage.reportPath=report/CoveragePackageReport*.xml
```

Comment:
Not generating coverage, so not needed

Reply:
to be fixed when test part is completed.